### PR TITLE
Deploy app using GCP Cloud Run

### PR DESCRIPTION
**Pull request by operte**
I decided to host the app on my personal GCP account using their free tier.
I have been learning about some GCP tools, so I wanted to explore the deployment possibilities as well :) 

As far as I could gather, there are two main options to deploy a stateless app such as ours on GCP:
- App Engine: we just need to provide a `requirements.txt` and App Engine will automatically set everything up. After deployment, the app that is always running.
- Cloud Run: we need to containerize the app ourselves. After deployment, the containers are booted up whenever there's a request. They timeout after ~5 min, not sure we can control the timeout in the free tier.

This wasn't as easy as I expected, namely due to:
- using `poetry` for dependency management: App Engine expects a `requirements.txt`, which we could generate manually from `poetry`, but which is an additional annoying step to do.
- the whole `geopandas`/`fiona`/`gdal` dependency chain doesn't work very well, and I think we must also install `gdal` using the system's package manager, and not just through Python's dependency manager.

After trying App Engine and failing, I went with Cloud Run. It took me some time to figure everything out but in the end it's quite easy to understand:
- There's a `Dockerfile` that configures the container. It uploads the source code to GCP, installs `gdal` using `apt-get`, installs our package using `poetry` and serves it using `gunicorn`
- There's a `.dockerignore` file which lists files that should not be uploaded to GCP.

The app is available [here](https://paris-bikes-wfiz3bgwsa-ew.a.run.app/). 

Booting up the container takes a bit of time, up to 1 min. Also loading all the data columns takes a bit of time so in the beginning the app doesn't feel too responsive, but after that it is fine.
This means there's certainly room for optimization. If someone sees something strikingly wrong with our code, let me know :p Otherwise I will leave it as is.

If you would like to check out the project on GCP, let me know and I'll try to add you to it as an admin. I'm just not sure that's supported with the free tier.

Solves #62

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,19 +3,22 @@
...
-- [Contributors and supporters](#contributors-and-supporters)
-    - [We thank](#we-thank)
-- [Project setup](#project-setup)
-    - [Dash application](#dash-application)
-    - [Development environment](#development-environment)
-    - [Data management](#data-management)
-- [Project management](#project-management)
-    - [Useful links](#useful-links)
-    - [Definition of Done](#definition-of-done)
+- ["Where to build new bicycle parking spots in Paris? Supporting data-driven decision-making with open data"](#where-to-build-new-bicycle-parking-spots-in-paris-supporting-data-driven-decision-making-with-open-data)
```
Comment (katoss) [inline README.md:6]: I think we agreed to remove the first two menu points (title and TOC) in an older PR. Did you re-add them on purpose ?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Use the official lightweight Python image.
```
Comment (katoss) [inline Dockerfile:1]: The comments in the dockerfile are really helpful, thanks for adding them

File: README.md [hunk 3]
```diff
@@ -29,15 +32,15 @@ And this is where CorrelAid comes in! *(You don't know CorrelAid? Have a [look h
...
+From July to December 2022, the team explored open data repositories, researched approaches of other cities, brainstormed, discussed, designed, and coded. The results can be found in this repository :)
 
 ## Paris Parking Demand Index
 
 The Paris Parking Demand Index is a web app that visualizes the number of existing bicycle parking spaces in the City of Paris in relation to metrics that might indicate demand for bicycle parking, such as the number of stores or people entering the metro.
 Aggregated at the IRIS level, the smallest unit of municipal infrastructure in France, this index helps determine how adequately areas are served in terms of parking facilities, while leaving flexibility as to which exact location they should be built.
 All data used in this project is from French open data portals.
 
-Have a look here at a hosted version of the Parking Demand Index *[Link will follow soon]*
+Have a look [here](https://paris-bikes-wfiz3bgwsa-ew.a.run.app/) at a hosted version of the Parking Demand Index (it might take up to a minute to load).
```
Comment (katoss) [inline README.md:43]: Good idea to add a note on loading times!

**Review by katoss — COMMENTED** [review]
Looks good to me as far as I can judge, thank you! I just had a minor comment on the readme

**Comment by operte** [comment]
Thanks for adjusting it! I don't know why your changes to the readme were reverted, I probably did something wrong 😅

**Pull request merged**